kubernetes_state/check.py

```diff
@@ -181,6 +180,13 @@ def _label_to_tag(self, name, labels, tag_name=None):
         else:
             return None
 
+    def _remove_job_tag(self, name):
+        """
+        Removes suffix of job names if they match -(\d{4,10})
+        """
+        pattern = "(-\d{4,10})"
```

Comment:
I'd go with `(-\d{4,10}$)` to make sure to match the end of the name, is that OK with you?